bin/cli.ml

```diff
@@ -53,6 +53,18 @@ let opam =
   let docv = "FILE" in
   Arg.(value & opt (some path_arg) None & info [ "opam" ] ~doc ~docv)
 
+let token =
+  let doc =
+    "The github token to use. The regular case is to have a locally store the \
+     github token that $(b,dune-release) can automatically use and to not set \
+     this option. This option is used to override the local configuration \
+     token for example as part of a Github Actions workflow where the github \
+     token is provided through an environment variable."
+  in
+  let docv = "FILE" in
+  let env = Arg.env_var "GITHUB_TOKEN" in
```

Comment:
Can you prefix it with `DUNE_RELEASE_` please to make sure we don't accidentally get another token?

Reply:
done